conf/application.conf

```diff
@@ -196,3 +196,57 @@ versioning {
   ]
 
 }
+
+api-config {
+    scopes {
+        "read:individuals-employments-laa-c1" {
+            fields = ["K", "L"]
+        }
+        "read:individuals-employments-laa-c2" {
+            fields = ["K", "L"]
+        }
+        "read:individuals-employments-laa-c3" {
+            fields = ["A", "K", "L"]
+        }
+        "read:individuals-employments-laa-c4" {
+            fields = ["A", "D", "E", "F", "G", "H", "I", "K", "L"]
+        }
+        "read:individuals-employments-hmcts-c2" {
+            fields = ["L"]
+        }
+        "read:individuals-employments-hmcts-c3" {
+            fields = ["L"]
+        }
+        "read:individuals-employments-hmcts-c4" {
+            fields = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "K", "L"]
+        }
+        "read:individuals-employments-lsani-c1" {
+            fields = ["A", "K", "L"]
+        }
+        "read:individuals-employments-lsani-c3" {
+            fields = ["A", "K", "L"]
+        }
+        "read:individuals-employments-nictsejo-c4" {
+            fields = ["A", "D", "E", "F", "G", "H", "I", "K"]
+        }
+    }
+    endpoints {
+        employments {
+            endpoint = "/individuals/employment/paye"
```

Comment:
Update to correct HAL link and ensure endpoint keys match those in the controllers `getEndPointScopes` invocation.